compiler-cli/src/run.rs

```diff
@@ -21,24 +23,71 @@ pub fn command(
     arguments: Vec<String>,
     target: Option<Target>,
     runtime: Option<Runtime>,
+    module: Option<String>,
     which: Which,
 ) -> Result<(), Error> {
+    // Validate the module to make sure it is a gleam module path
+    match &module {
+        Some(module_name) => {
+            // TODO: Check if this can be replaced with a function that
```

Comment:
This regex is roughly what you want, though it also checks that there's `.gleam` at the end.

https://github.com/gleam-lang/gleam/blob/67c2d94a2afbfe14ba9075c7761cb6fd1a9c7eaf/compiler-cli/src/fs.rs#L301-L308

Reply:
Does a `is_valid_module_name` function belong in this file? Or should I create a new function/regex in `compiler-cli/run.rs`?

Reply:
Since [this command](https://github.com/gleam-lang/gleam/pull/2068#issuecomment-1467889908) i think is pretty likely to be a feature in the future im going to use regex validation that meant specifically for this one thing.